taxi/views.py

```diff
@@ -60,10 +62,19 @@
 
 class CarDetailView(LoginRequiredMixin, generic.DetailView):
     model = Car
+    template_name = "taxi/car_detail.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        user = self.request.user
+        car = self.get_object()
+        context["is_driver"] = user in car.drivers.all()
+        return context
 
 
 class CarCreateView(LoginRequiredMixin, generic.CreateView):
     model = Car
+    form_class = CarCreateForm
     fields = "__all__"
```

Comment:
In `CarCreateView`, specifying both `form_class` and `fields` is redundant. Since you are using `CarCreateForm`, you should remove `fields = "__all__"`.